docs/rfcs/024-ingress-egress-metrics.md

```diff
@@ -0,0 +1,64 @@
+# Ingress-Egress Metrics
+
+# Goals
+* Track ingress/egress traffic directly from a compute node, which may be sending stuff due to 
+  a custom extension.
+* Lay the groundwork for further traffic management on the compute node. 
+
+# Background
+As we move towards allowing custom extensions on the compute node, we need to cope with the fact
+that these extensions are untrusted. They could send or receive things from the open internet or
+even try to send bogus requests around on the internal network. Although we currently use the proxy
+to measure incoming/outgoing traffic, custom extensions will potentially be able to interact with
+the outside without involving the proxy. This has a high potential for abuse, so we need some ability
+to measure and lock down usage. This RFC concerns itself with the first part (measuring).
+
+# Possible Implementations
```

Comment:
For traffic monitoring there are quite a few existing solutions in the Kubernetes ecosystem, as that's an essential part of observability.
* https://kubernetes.io/docs/tasks/debug/debug-cluster/resource-metrics-pipeline/
* https://cilium.io/
* https://px.dev/

To mention a few. Also commercial solutions do exist. I'm not sure it makes that much sense to take an opinionated approach over allowing to build on top of existing tools.